pkg/registry/clusterrolebinding/policybased/storage.go

```diff
@@ -0,0 +1,59 @@
+/*
```

Comment:
Correct. AuthorizationRuleResolver requires something that can get/list roles/rolebindings.